Tasks/MSBuildV1/MSBuild.ps1

```diff
@@ -24,6 +26,16 @@ try {
     [string]$msBuildVersion = Get-VstsInput -Name MSBuildVersion
     [string]$msBuildArchitecture = Get-VstsInput -Name MSBuildArchitecture
 
+    $msbuildTelemetry = [PSCustomObject]@{
+        MSBuildVersion = $msBuildVersion
+        MSBuildArguments = $msBuildArguments
+        MSBuildLocation = $msBuildLocation
```

Comment:
Does it mean to include resolved location also - instead of the input value only?

Reply:
msBuildLocation doesn't contain resolved path, it is just a value from input